src/views/Home/components/Milestones/milestones.scss

```diff
@@ -1,3 +1,59 @@
-/* stylelint-disable-line no-empty-source */
+@use "~stylesheets/typography";
+@use "~stylesheets/responsiveness";
 
-/* Delete these lines when you begin development */
+.Milestones {
+  overflow: auto;
+  margin: 0;
+  min-height: 500px;
+  background: url("~images/components/milestones/background.png") repeat-y, #1a1542;
+  background-size: 100% auto;
+
+  &__title {
+    @include typography.title-text;
+
+    margin: var(--extra-large-spacer) 0;
+    text-shadow: 3px 3px #0085ff, -2px -2px rgba(255, 0, 0, 0.57);
```

Comment:
We might have to create variables for the text shadows, but I'm not too sure about that.